starter-code/pseudocode.txt

```diff
@@ -37,8 +37,8 @@ func countDivisionsByTwo(var n as int)
     return count
 end func
 
-Running time: 
-Explanation: 
+Running time: Big O(n)
+Explanation: It’s linear operation that takes in an int and returns how much how many times it is divisible by 2 while the number being divided next is greater than or equal to 1 which depends on how large or small of a number the input is
```

Comment:
This one is actually O(log(n)), because the loop doesn't run for n iterations, only log(n) iterations. This is because rather than do `i--` each time, it does `i/2`, and when you keep dividing the work in half like that, you end up with log(n) iterations.